ui/pages/confirmations/hooks/useConfirmationRecipientInfo.ts

```diff
@@ -11,11 +11,11 @@ function useConfirmationRecipientInfo() {
   const currentConfirmation = useSelector(
     currentConfirmationSelector,
   ) as Confirmation;
+  const allAccounts = useSelector(accountsWithSendEtherInfoSelector);
```

Comment:
Sweet. looks like this fixes the `Rendered more hooks than during the previous render` bug 👍🏼 

ref: https://github.com/orgs/MetaMask/projects/78/views/1?pane=issue&itemId=61449112